.evergreen/run-alpine-fle-tests.sh

```diff
@@ -9,5 +9,10 @@ set -o xtrace
 export npm_config_cache=$(pwd)/.cache
 npm install
 
+# Fix to point at the drivers tools pems installed in src.
+export CSFLE_TLS_CA_FILE=$(pwd)/drivers-evergreen-tools/.evergreen/x509gen/ca.pem
```

Comment:
Instead of re-writing these paths - could we instead just run the container tests in a directory that is the same path as the host machine?  Something like:

```
   docker --debug run \
        --platform linux/$LINUX_ARCH \
        -e MONGODB_URI=${MONGODB_URI} -e MONGOCRYPTD_URI=${MONGOCRYPTD_URI} \
        --volume $(pwd):$(pwd)$ -w $(pwd)$ \
        --network host \
        --entrypoint bash \
        $IMAGE_TAG \
        '.evergreen/run-alpine-fle-tests.sh'
```

Also - this is a dumb question because I wrote these scripts, but where do these variables currently get set in the container tests?  I don't see them exported anywhere in run-alpine-fle-tests.sh

Reply:
These variables are set by the tools script: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/setup-secrets.sh which is setting an absolute path to the pems. I'm not sure changing where we run the container tests is going to affect anything.

Reply:
If we mount the docker container at the same path (inside the container) as the driver is on the host, the certificate paths will work in either the host or the container.